tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/ClientSideEncryptionTestRunner.cs

```diff
@@ -151,9 +151,18 @@ protected override bool TryConfigureClientOption(MongoClientSettings settings, B
             return true;
         }
 
-        protected override void VerifyCollectionData(IEnumerable<BsonDocument> expectedDocuments)
+        protected override void VerifyCollectionData(
+            IEnumerable<BsonDocument> expectedDocuments,
+            string databaseName,
+            string collectionName,
+            ReadConcern readConcern = null)
```

Comment:
I don't think readConcern is needed here or in the base class.

Reply:
done